backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -1397,27 +1398,33 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
             val thisPtr = evaluateExpression(value.receiver!!)
             return functionGenerationContext.loadSlot(
                     fieldPtrOfClass(thisPtr, value.symbol.owner), value.descriptor.isVar())
-        }
-        else {
+        } else {
             assert (value.receiver == null)
             val ptr = context.llvmDeclarations.forStaticField(value.symbol.owner).storage
             return functionGenerationContext.loadSlot(ptr, value.descriptor.isVar())
         }
     }
 
     //-------------------------------------------------------------------------//
+    private fun needMutationCheck(expression: IrExpression): Boolean {
+        // For now we omit mutation checks on immutable types, as this allows initialization in constructor
+        // and it is assumed API doesn't allow to change them.
```

Comment:
> it is

its?

Reply:
Hmm, not really.

Reply:
> and it is assumed API

Then "that" is missing between "assumed" and "API".